Angular/src/app/login/login.component.html

```diff
@@ -0,0 +1,52 @@
+<div class="limiter">
+    <div class="container-login100">
+        <div class="wrap-login100 p-t-85 p-b-20">
+            <div class="login100-form validate-form">
+                <span class="login100-form-title">
+                    <img src="https://www.kainos.com/assets/header/Kainos-Logo-841a14fb110ca1f8e9ccee8f06c6cb34baa568b8a004b21d400ed2c4e5fbbb1b.png">
```

Comment:
I'd store this image as an asset in your frontend folder (under `Angular/src/assets`), and reference its path here in the HTML. It's better than having a direct URL to the image

Reply:
no worries sorted